node/core/dispute-coordinator/src/real/mod.rs

```diff
@@ -542,16 +571,25 @@ async fn handle_incoming(
 		},
 		DisputeCoordinatorMessage::RecentDisputes(rx) => {
 			let recent_disputes = overlay_db.load_recent_disputes()?.unwrap_or_default();
+			tracing::debug!(target: DEBUG_LOG_TARGET, "Recent Disputes {:?}.", recent_disputes,);
 			let _ = rx.send(recent_disputes.keys().cloned().collect());
 		},
 		DisputeCoordinatorMessage::ActiveDisputes(rx) => {
+			// TODO (ladi): Shouldn''t this load active disputes?
```

Comment:
I think there's only 'recent disputes' on disk, and then this filters them to be only active